compiler/rustc_middle/src/lib.rs

```diff
@@ -47,10 +47,8 @@
 #![feature(extract_if)]
 #![feature(file_buffered)]
 #![feature(if_let_guard)]
-#![feature(intra_doc_pointers)]
```

Comment:
Might need this because the doc comment uses this?

Reply:
Oh, good catch. I had no idea what was causing that error.